source/draw/CreateAlphaByteString.ooc

```diff
@@ -32,9 +32,12 @@ CreateAlphaByteString: class {
 			ipbuffer pointer[i] = buf pointer[(i * 4) + 3]
 		ip = ipbuffer pointer as Int*
 		if (ipbuffer size > 3)
-			imageArray += ip[0] toString()
-		for (i in 1 .. ipbuffer size / 4)
-			imageArray += "," + (i % 32 == 0 ? "\n" : " ") + ip[i] toString()
+			imageArray = imageArray >> ip[0] toString()
+		for (i in 1 .. ipbuffer size / 4) {
+			intString := ip[i] toString()
+			imageArray = imageArray >> "," >> (i % 32 == 0 ? "\n" : " ") >> intString
```

Comment:
Does this part `imageArray >> "," >> (i % 32 == 0 ? "\n" : " ")` looks OK with valgrind ? I never know when a string like that is a static literal and I try to free it.